notebook/static/notebook/js/actions.js

```diff
@@ -423,6 +415,21 @@ define(function(require){
                 env.notebook.show_command_palette();
             }
         },
+        'toggle-toolbar':{
+            help: 'hide/show the toolbar',
+            handler : function(env){
+                $('div#maintoolbar').toggle();
```

Comment:
Oh, please no, don't use jquery here. I've made effort for `env` to be accessible. This is the only thing in the file that use jQuery.